workers/limiter/tests/index.test.ts

```diff
@@ -365,6 +368,48 @@ describe('Limiter worker', () => {
         data: expect.any(String),
       });
     });
+
+    test('Should block projects if workspace has isBlocked field with value true', async (done) => {
```

Comment:
describe specification please, not a code. When the Workspace has the `isBlocked` field?